src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--tue calendar--month-length--28">
+      <div class="calendar__days calendar__days--day--1"></div>
```

Comment:
these `day` divs should only have `calendar__days` class
```suggestion
      <div class="calendar__days"></div>
```